src/main/java/org/dinosaur/foodbowl/domain/post/entity/Category.java

```diff
@@ -36,6 +38,15 @@ private Category(Long id, CategoryType categoryType) {
         this.categoryType = categoryType;
     }
 
+    public static Category from(String categoryName) {
+        try {
+            CategoryType categoryType = CategoryType.valueOf(categoryName);
+            return from(categoryType);
+        } catch (IllegalArgumentException e) {
+            throw new FoodbowlException(ErrorStatus.CATEGORY_NOT_FOUND);
+        }
+    }
+
```

Comment:
CategoryType에게 문자열에 맞는 카테고리를 찾는 역할을 줄 수도 있을 것 같습니다 !
큰 차이는 없어서 ㅎㅎ 원하는 스타일로 사용하시면 좋을 것 같네용 😎

```suggestion
    public static Category from(String categoryName) {
        CategoryType categoryType = CategoryType.of(categoryName);
        return new Category(categoryType.id, categoryType);
    }
    
    public enum CategoryType {

        전체(1L),
        카페(2L),
        한식(3L),
        양식(4L),
        일식(5L),
        중식(6L),
        치킨(7L),
        분식(8L),
        해산물(9L),
        샐러드(10L);

        private final Long id;

        CategoryType(final Long id) {
            this.id = id;
        }

        public Long getId() {
            return id;
        }

        public static CategoryType of(String category) {
            return Arrays.stream(values())
                    .filter(categoryType -> categoryType.name().equals(category))
                    .findAny()
                    .orElseThrow(() -> new FoodbowlException(ErrorStatus.CATEGORY_NOT_FOUND));
        }
    }
```